packages/warehouses/src/warehouseClients/BigqueryWarehouseClient.ts

```diff
@@ -256,4 +259,15 @@ export class BigqueryWarehouseClient implements WarehouseClient {
     getEscapeStringQuoteChar() {
         return '\\';
     }
+
+    getMetricSql(sql: string, metric: Metric) {
+        switch (metric.type) {
+            case MetricType.PERCENTILE:
```

Comment:
so all metric types we have so far are the same for all warehouses (getDefaultMetricSql) , but this PERCENTILE is different per warehouse, right ?

Reply:
correct